docs/advanced_topics/icons.md

```diff
@@ -0,0 +1,90 @@
+(icons)=
+
+# Icons
+
+Wagtail comes with an icon set. The icons are used throughout the admin interface.
+
+Elements that use icons are:
+
+- [Register Admin Menu Item](register_admin_menu_item)
+- [Clientside components](extending_clientside_components)
+- [Rich text editor toolbar buttons](extending_the_draftail_editor)
+- [ModelAdmin menu](modeladmin_menu_icon)
+- [Streamfield blocks](custom_streamfield_blocks)
+
+This document describes how to choose and add icons.
+
+## Available icons and their names
+
+Icons are registered in `wagtail/admin/wagtail_hooks.py`. 
+The _filename_ without the extension is the icon name.
+
+Enable the [styleguide](styleguide) to view icons and their names.
+
+[//]: # (The code is included to present an up-to-date icon name list)
+
+```{eval-rst}
```

Comment:
This is cool! 👏

https://wagtail--6028.org.readthedocs.build/en/6028/advanced_topics/icons.html#available-icons-and-their-names

Any chance the whole block can we wrapped in `<details>` or similar? it is fairly large and takes a fair bit of scrolling to get the core of the doc

Reply:
Thanks, I like it too, however... I linked to the source code, and like a simple link even better.